File: Programmers/Lv0/문자개수세기.cpp
#include <string>
#include <vector>
#include <algorithm>
using namespace std;
vector<int> solution(string my_string) {
    vector<int> answer(52, 0);
    for (char ch : my_string) {
        if (islower(ch))
            answer[int(ch - 'a') + 26] += 1;
        else
            answer[int(ch - 'A')] += 1;
    }
    return answer;
}

File: Programmers/Lv0/문자개수세기.py
def solution(my_string):
    answer = [0] * 52
    for ch in my_string:
        if 'a' <= ch <= 'z':
            answer[(ord(ch) - ord('a')) + 26] += 1
        else:
            answer[ord(ch) - ord('A')] += 1
    return answer